gradle/swagger.gradle

```diff
@@ -0,0 +1,29 @@
+/*
+ * Plugin that provides API-first development using OpenAPI-generator to
+ * generate Spring-MVC endpoint stubs at compile time from an OpenAPI definition file
+ */
+apply plugin: "org.openapi.generator"
```

Comment:
remove this file, we have no api first trash in this microservice

Reply:
done